doc/advanced-configuration/outgoing-connections.md

```diff
@@ -213,20 +213,21 @@ A connection pool to ElasticSearch can be configured as follows:
 
 ```erlang
 {outgoing_pools, [
- {elastic, global, elasticsearch, [], [{host, "localhost"}]}
+ {elastic, global, default, [], [{host, "localhost"}]}
```

Comment:
I would add an explicit note that currently only one `elastic` pool is supported.